guides/common/modules/proc_creating-a-remote-file-repository.adoc

```diff
@@ -25,24 +25,25 @@ endif::[]
 
 To create a file type repository in a remote directory, complete the following procedure:
 
-. On your remote server, ensure that the Server and {project-client-name} repositories are enabled.
 ifdef::satellite[]
+. On your machine, ensure that the right repositories are enabled.
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# subscription-manager repos --enable={RepoRHEL7Server} \
---enable={project-client-RHEL7-url}
+# subscription-manager repos --enable={RepoRHEL8BaseOS} \
+--enable={RepoRHEL8AppStream} \
+--enable={RepoRHEL8ServerSatelliteUtils}
 ----
 endif::[]
 . Install the Pulp Manifest package:
 +
-[options="nowrap" subs="+quotes"]
+[options="nowrap" subs="+quotes,attributes"]
 ----
-# yum install python3-pulp_manifest
+# {package-install} python3-pulp_manifest
 ----
 . Create a directory that you want to use as the file type repository in the HTTP server's public folder:
-+
 [options="nowrap" subs="+quotes"]
++
```

Comment:
this does not look right to me. IMHO there is no change necessary in line 44 and 46.

Reply:
M not getting what do you want to suggest. There is nothing changed in in line 44 and 46.

Reply:
It should be the following order: bullet point, "+", code block specification, "---", code, and "----".